### PR TITLE
mpdris2: add module

### PR DESCRIPTION
**Pull request by pjones**

### CODE REVIEW

File: modules/misc/news.nix [hunk 1]
```diff
@@ -991,6 +991,13 @@ in
           A new module is available: 'programs.keychain'.
         '';
       }
+
+      {
+        time = "2019-02-20T15:37:15+00:00";
+        message = ''
+          A new service is available: 'services.mpDris2'.
```
Comment (rycee) [inline modules/misc/news.nix]: I think `mpdris2` is better since it is consistent with the package and file names.

Comment (pjones) [reply modules/misc/news.nix]: Fixed.

**Comment by rycee** [comment]
I pushed a commit to your branch with a few proposed changes to the module. Please have a look and see if you agree. It's mostly cosmetics but the change to use `defaultText` was a bit more important since the documentation otherwise would depend on the configuration.

**Comment by pjones** [comment]
Looks good to me.  Thanks for cleaning it up.

**Comment by rycee** [comment]
Great! Rebased to master in f07510e2b63075a5e334e603f666c5c5838563ce. Thanks for the contribution 🙂

**Pull request closed without merging**